addons/web/static/src/core/datetime/datetimepicker_service.js

```diff
@@ -268,7 +268,7 @@ export const datetimePickerService = {
                     const convertFn = (operation === "format" ? formatters : parsers)[type];
                     try {
                         return [
-                            convertFn(value, { format: hookParams.format, tz: pickerProps.tz }),
+                            convertFn(value, { format: hookParams.format, tz: pickerProps.tz, showSeconds: pickerProps.showSeconds }),
```

Comment:
be aware that your code must be linted/prettified (you can enable the tooling via ./addons/web/tooling/enable.sh)